packages/runtime/src/slate/index.ts

```diff
@@ -1,10 +1,11 @@
-export * from './TypographyPlugin'
+export { type NodeJSON, type ValueJSON, type InlineJSON, type MarkJSON } from '@makeswift/controls'
```

Comment:
We may not need to expose these anymore, will confirm

Reply:
Confirmed - not used in the builder anymore after we replaced the duplicated RichText copy functionality. Removing.